### ISSUE TITLE
"Bed Mode"

### ISSUE THREAD

**Issue by Frooxius** [Frooxius/NeosPublic#225]
Recently 0utsider has suggested another mode in addition to seated - bed mode. This would automatically orient the head forwards, regardless of where is the user looking vertically, allowing for social interactions in any position.

This could be used as comfort mode when lying on the back leaning back in a chair, but would also help with accessibility, allowing bedridden or otherwise impaired users to use Neos more easily and participate in social interactions.

Any thoughts?

**Comment by SecretHippo1** [comment]
I don’t only want to see this, but I need to see this 😬 Preferably with the launch of the Quest for obvious reasons. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Karel64** [comment]
Excellent idea and opportunity to bring Neos to people with long term health problems or recovering. We should make sure to always note that it's meant for people with health issues when mentioning it. Could turn into a PR controversy otherwise.

**Comment by Simulacron3** [comment]
Great idea, but I would have concerns about disorientation effects and motion sickness. Experimentation needed.

**Comment by Karel64** [comment]
That's an excellent point. Especially new VR users sometimes don't realize they should immediately take off headset when/if starting too feel sick. They sometimes try to endure and get really sick. There has to be some note about that and we have to mention it when in contact with the people demoing or providing help in these scenarios.

**Comment by SecretHippo1** [comment]
I volunteer as tribute.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by VixusFoxy** [comment]
Same, this is a wonderful idea and aside from an inner-ear-confusion state that will take time to overcome I can certainly see the utility in this mode. :3

**Comment by TiagoTiago** [comment]
What if instead of locking the view direction, you had a button to define the current head orientation as "forward" and just apply that offset to any head orientation?

**Comment by shiftyscales** [comment]
It's been discussed before, but I believe that was the planned implementation, @TiagoTiago. You activate the option, and it will apply an offset to your playspace rotation based on your head's position when it was activated.

**Comment by Frooxius** [comment]
Yes, that's what the bed mode is planned to be. When you activate it, whatever direction you're facing will become the new "forward".

**Comment by kulzae** [comment]
Could possibly avoid PR issues by timing this with integration for the "half dive" headset that is meant to be used while lying down (https://diver-x.jp/halfdive/en/) could change name from bed mode to reclined mode (standing, sitting, reclined)

**Comment by epicEaston197** [comment]
> Could possibly avoid PR issues by timing this with integration for the "half dive" headset that is meant to be used while lying down (https://diver-x.jp/halfdive/en/) could change name from bed mode to reclined mode (standing, sitting, reclined)

well this aged really poorly as the half dive headset is canceled
![image](https://user-images.githubusercontent.com/76523692/206085230-a1af7c8d-ba36-4674-8394-738bcd831783.png)
